.github/workflows/node.js.yml

```
@@ -13,18 +10,44 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [22.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
-      - uses: actions/checkout@v4
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

```suggestion
          fetch-depth: 1
```
0 fetches all the history while 1 will fetch only the latest commit reducing network load. The default value is 1, Is there a specific reason we are changon it to 0 ?

---

removing depth as we don't need anything from previous commits.